### PR TITLE
Setup tranaction fee

### PR DESCRIPTION
**Pull request by nocmar**

### CODE REVIEW

File: example/src/App.tsx [hunk 1]
```diff
@@ -1,82 +1,97 @@
...
-import 'starterra-tool-dapp/dist/index.css'
+import {
+  WalletProvider,
+  Wallet,
+  Tokens,
+  NetworkInfo
+} from 'starterra-tool-dapp'
 
+const avaliableNetworks: Record<string, NetworkInfo> = {
+  0: {
```
Comment (Mskalba) [inline example/src/App.tsx:11]: Should it be 0 or 1? Maybe key like: "tequila"/ "columbus" would be more descriptive?

Comment (nocmar) [reply example/src/App.tsx:11]: I agree, and I put it with "testnet", "mainnet" before, unfortunately this format is required by wallet provider. In some point they flatten it and expected indexes. The read only mode wasn't working when the network name was as key. I changed it today to fix it.

Comment (Mskalba) [reply example/src/App.tsx:11]: Omg :D i get it.

**Review by Mskalba — APPROVED** [review]
Good job.

**Pull request merged**